### PR TITLE
Adding Data Request template draft. Initial guide for DataTeam

### PR DESCRIPTION
**Pull request by wildintellect**

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/data_request.md [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+name: Data Request
+about: Suggest a data set for ingestion in the platform
+title: ''
+labels: 'MSFC'
+assignees: ''
+
+---
+
+**Dataset Name**
+What is the name of the data requested.
```
Comment (abarciauskas-bgse) [inline .github/ISSUE_TEMPLATE/data_request.md:11]: Nit - this is a question should it end in a question mark?

File: .github/ISSUE_TEMPLATE/data_request.md [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+
+**Dataset Description**
+Short description of the data.
```
Comment (abarciauskas-bgse) [inline .github/ISSUE_TEMPLATE/data_request.md:14]: Here and elsewhere in this file I think either this should be a directive:

> Please provide a short description of the data.

or the prompt should not end in a period.

File: README_DataTeam.md [hunk 3]
```diff
@@ -0,0 +1,17 @@
+# Guide for the Data Team
+
+This is a short Readme specific to Data Team (MSFC) tasks, and usage of Zenhub.
+
+## Zenhub
+
+To make full use of Zenhub you need to install the browser extension. This will allow you to the Workspaces under the Zenhub tab in each repo.
```
Comment (abarciauskas-bgse) [inline README_DataTeam.md:7]: Should we make a note that the extension is only available in certain browsers - I think you mentioned it's only available for Firefox and Chrome.

File: README_DataTeam.md [hunk 4]
```diff
@@ -0,0 +1,17 @@
...
+
+## Issues
+
+You can file tickets either from the Zenhub DataTeam workspace, or on the respective github repo the issue relates to. The workspace board will aggregate the tickets for viewing in a single board.
+
+If the issue is general to the platform, you aren't sure which repo it belongs in, or is coming from a platform **user** file it under the Zenhub repo, and add appropriate labels. If it concerns the data team label **MSFC**, use as many labels as seems relevant.
+
+## Data Requests
+
+End users for now will be encouraged to file tickets in the [Zenhub](https://github.com/MAAP-Project/ZenHub) repo. This includes requests for upgrades to, or new data. For data requests, users should select the data request template.
```
Comment (abarciauskas-bgse) [inline README_DataTeam.md:17]: suggested wording:

>This includes requests for upgrades to existing data or new data.

Comment (abarciauskas-bgse) [reply README_DataTeam.md:17]: Can we link to the data request template?

**Review by abarciauskas-bgse — COMMENTED** [review]
Minor comments otherwise looks good, thanks @wildintellect !

File: .github/ISSUE_TEMPLATE/data_request.md (hunk 2 above)
Comment (kbugbee1) [reply .github/ISSUE_TEMPLATE/data_request.md:14]: Plus one to Aimee's comment.

Comment (kbugbee1) [reply .github/ISSUE_TEMPLATE/data_request.md:14]: Being Southern, I generally prefer to use the word 'Please' :)

**Review by kbugbee1 — COMMENTED** [review]
I agree with Aimee's comments. Otherwise, this looks good to me.

File: README_DataTeam.md (hunk 4 above)
Comment (wildintellect) [reply README_DataTeam.md:17]: Yes, once it exists should be
https://github.com/MAAP-Project/ZenHub/issues/new?template=data_request.md
Will add that to text, might need to check and adjust after merge.

File: README_DataTeam.md (hunk 3 above)
Comment (wildintellect) [reply README_DataTeam.md:7]: https://help.zenhub.com/support/solutions/articles/43000507578-installing-the-zenhub-extension-for-cloud

File: .github/ISSUE_TEMPLATE/data_request.md (hunk 2 above)
Comment (wildintellect) [reply .github/ISSUE_TEMPLATE/data_request.md:14]: Is it a prompt or a replace this text with actual value?

**Pull request merged**